src/js/engagement_view/src/components/reusableComponents/drawerList.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from "react";
+
+import { NavLink } from "react-router-dom";
+
+import List from "@mui/material/List";
+import ListItemButton from "@mui/material/ListItem";
+import ListItemIcon from "@mui/material/ListItemIcon";
+import ListItemText from "@mui/material/ListItemText";
+
+import BubbleChartIcon from "@mui/icons-material/BubbleChart";
+import PolicyIcon from "@mui/icons-material/Policy";
+import ExtensionIcon from "@mui/icons-material/Extension";
+
+import { useStyles } from "../styles/analyzersAndGeneratorsStyles";
+
+export const NavListItems = () => {
+    const classes = useStyles();
+
+    return (
+        <>
+            <List className={classes.drawer}>
+                <ListItemButton>
+                    <NavLink
+                        to="/analyzers"
+                        className={classes.navLink}
+                        style={({ isActive }) => ({
+                            color: isActive ? "#1A76D2" : "#C6D1E7",
+                        })}
+                    >
```

Comment:
It looks like react-router offers a more robust way to do this:

https://v5.reactrouter.com/web/api/NavLink/activeclassname-string

by default, when a NavLink becomes active, its `class` becomes `className + ".active"`

so I think you can just do this in the styles file

```
        navLink: {
            textDecoration: "none",
            color: theme.palette.info.contrastText,
            "& .active": {
              color: theActiveColorGoesHere
            }
        },
```

I _believe_ the & is how you do `.navlink.active` in this CSS library, but I'm not entirely sure.

Reply:
I think the above method was right for v5 of react router, but according to the linked docs for v6 (which we just started using) I think the docs want us to style active links with functions. Happy to change this, but according to the following paragraph, I think we're supposed to be using inline styles, but could totally be reading this wrong. Happy to change if necessary. 

```
In React Router v6, activeStyle will be removed and you should use the function style to apply inline styles to either active or inactive NavLink components.
```